content/intro-to-storybook/angular/en/data.md

```diff
@@ -5,23 +5,23 @@ description: 'Learn how to wire in data to your UI component'
 commit: '0bf4edf'
 ---
 
-So far we created isolated stateless components –great for Storybook, but ultimately not useful until we give them some data in our app.
+So far, we have created isolated stateless components –great for Storybook, but ultimately not helpful until we give them some data in our app.
 
-This tutorial doesn’t focus on the particulars of building an app so we won’t dig into those details here. But we will take a moment to look at a common pattern for wiring in data with container components.
+This tutorial doesn’t focus on the particulars of building an app, so we won’t dig into those details here. But we will take a moment to look at a common pattern for wiring in data with container components.
 
 ## Container components
 
-Our `TaskList` as currently written is “presentational” (see [this blog post](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)) in that it doesn’t talk to anything external to its own implementation. To get data into it, we need a “container”.
+Our `TaskList` component as currently written is “presentational” (see [this blog post](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)) in that it doesn’t talk to anything external to its own implementation. To get data into it, we need a “container”.
```

Comment:
The first thing in this article is the author denouncing it as a best practice. Should we really be linking to it? Or advocating the "container vs. presentational" split in the first place?

Reply:
Yeah, the "container" phrasing really should be dropped at this point.  We removed that term from the Redux docs completely when I rewrote our tutorials last year.  The real question is "assuming my component needs some data, how do I make that accessible in the story?"

Reply:
I'll remove the link for now. Once 6.4 is out I'm planning on revisiting this section and the Screen section and providing more concrete examples. @markerikson appreciate the feedback and hope you don't mind but once that pull request is up I'll loop you in for a review from a redux toolkit perspective. Sounds good?